source/nodejs/adaptivecards-site/package.json

```diff
@@ -1,56 +1,56 @@
 {
```

Comment:
this file seems to have been modified, for some reason tabs were changed to spaces